src/multichoice/index.ts

```diff
@@ -260,77 +260,81 @@ let exportMultiChoice;
         case "ArrowUp": {
           event.preventDefault();
           if (this.items.length === 0) break;
-          this._itemsEl.forEach((itemEl: HTMLDivElement, number: number) => {
-            if (
-              itemEl.classList.contains(
-                "kuc-multi-choice__group__menu__highlight"
-              )
-            ) {
-              itemEl.classList.remove(
-                "kuc-multi-choice__group__menu__highlight"
-              );
-              highLightNumber = number - 1;
-            }
-          });
-          highLightNumber =
-            highLightNumber <= -1 ? this._itemsEl.length - 1 : highLightNumber;
-
-          const currentItemEl = this._itemsEl[highLightNumber];
-          currentItemEl.classList.add(
-            "kuc-multi-choice__group__menu__highlight"
-          );
-
-          this._setActiveDescendant(currentItemEl.id);
+          highLightNumber = this._actionHighlightPrevMenuItem(highLightNumber);
```

Comment:
I think we don't need `highLightNumber` variable, It seems to always contain the value = 0.
Please consider to remove it.